packages/legend-application-studio/style/components/editor/uml-editor/_uml-element-editor.scss

```diff
@@ -16,6 +16,42 @@
 
 @use 'mixins' as *;
 
+.uml-element-dnd-placeholder {
```

Comment:
let's follow `BEM` and rename these classes to `uml-element-editor__dnd-*`

Reply:
👍